src/components/Info/styles.styl

```diff
@@ -0,0 +1,24 @@
+@require 'settings/palette.styl'
+@require 'tools/mixins'
+
+.infos
+    background paleGrey
+    display flex
+    align-items center
+    background-color #fff2f2
+
+.infos--icon
+    width rem(16)
```

Comment:
why you don't use 1rem directly?

Reply:
If you remove it you can remove `@require 'tools/mixins'`

Reply:
It's a copy/paste from https://github.com/cozy/cozy-ui/blob/master/react/Infos/styles.styl. I don't really know why we use this helper.